### ISSUE TITLE
FS-600

### ISSUE THREAD

**Issue by mindlight73** [Gissio/radpro#9]
Do you still plan to support FS-600 and if so, when in time?
I just got a FS-600 and it bugs me that there is a USB-C port but the device lacks capability of reporting anything through it.

If you need pictures of the internals, please let me know.

**Comment by Gissio** [comment]
FS-600 support is around the corner. Hang tight!

**Comment by Gissio** [comment]
Rad Pro 2.0 Beta 11 is out!

https://github.com/Gissio/radpro/releases

**Comment by Gissio** [comment]
I forgot to mention. It has support for the Bosean FS-600. Were you able to try it?

**Comment by mindlight73** [comment]
Nope. I saw it today. I don't have the serial USB thingy to flash it so I will purchase that tonight and I will open it up and do some soldering next week.... Pray for me 😁

**Comment by Gissio** [comment]
Rad Pro 2.0 Beta 14 is out!

https://github.com/Gissio/radpro/releases

**Comment by mindlight73** [comment]
Just wanted to add that I finally got a STM32. Yeah, it took a year...
I just flashed my Bosean FS-600. Worked like a charm!

**Comment by Gissio** [comment]
I'm glad it worked!
